js-exercises/chunk-array-in-groups/chunkArrayInGroups.js

```diff
@@ -0,0 +1,12 @@
+function chunkArrayInGroups(array, chunkSize) {
+  // check for no argument
+  if (!array) return [];
+  // if size of array is less than or equal to chunk size pass the array
```

Comment:
@jaypee01 As mentioned in the session, these are redundant comments. we can remove them 👍